se_apps/include/interface.h

```diff
@@ -109,526 +188,640 @@ static const float _focal_x = SceneK[0][0];
-							* sizeof(unsigned short int), SEEK_CUR);
-			expected_size += 1 + newImageSize[0] * newImageSize[1];
-		}
+      if (depthMap) {
+        total += fread(depthMap, sizeof(unsigned short int),
+            newImageSize[0] * newImageSize[1], _pFile);
+        expected_size += 1 + newImageSize[0] * newImageSize[1];
+      } else {
+        total += newImageSize[0] * newImageSize[1];
+        fseeko(_pFile,
+            newImageSize[0] * newImageSize[1]
+            * sizeof(unsigned short int), SEEK_CUR);
+        expected_size += 1 + newImageSize[0] * newImageSize[1];
+      }
 
 #ifdef LIGHT_RAW // This LightRaw mode is used to get smaller raw files
 
-		if (raw_rgb) {
-			raw_rgb[0].x = 0;
-		} else {
-		}
+      if (raw_rgb) {
+        raw_rgb[0].x = 0;
+      } else {
+      }
 
 #else
-		total += fread(&(newImageSize), sizeof(newImageSize), 1, _pFile);
-
-		if (raw_rgb) {
-			total += fread(raw_rgb, sizeof(uchar3),
-					newImageSize[0] * newImageSize[1], _pFile);
-			expected_size += 1 + newImageSize[0] * newImageSize[1];
-		} else {
-			total += newImageSize[0] * newImageSize[1];
-			fseeko(_pFile, newImageSize[0] * newImageSize[1] * sizeof(uchar3),
-					SEEK_CUR);
-			expected_size += 1 + newImageSize[0] * newImageSize[1];
-		}
+      total += fread(&(newImageSize), sizeof(newImageSize), 1, _pFile);
+
+      if (raw_rgb) {
+        total += fread(raw_rgb, sizeof(uchar3),
+            newImageSize[0] * newImageSize[1], _pFile);
+        expected_size += 1 + newImageSize[0] * newImageSize[1];
+      } else {
+        total += newImageSize[0] * newImageSize[1];
+        fseeko(_pFile, newImageSize[0] * newImageSize[1] * sizeof(uchar3),
+            SEEK_CUR);
+        expected_size += 1 + newImageSize[0] * newImageSize[1];
+      }
 #endif
 
-		if (total != expected_size) {
-			std::cout << "End of file" << (total == 0 ? "" : "(garbage found)")
-					<< "." << std::endl;
-			return false;
-		} else {
-			return true;
-		}
-	}
-
-	inline void restart() {
-		_frame = -1;
-		rewind(_pFile);
-
-	}
-
-	inline bool readNextDepthFrame(float * depthMap) {
-
-		unsigned short int* UintdepthMap = (unsigned short int*) malloc(
-				_size.x * _size.y * sizeof(unsigned short int));
-		bool res = readNextDepthFrame(NULL, UintdepthMap);
-
-		for (unsigned int i = 0; i < _size.x * _size.y; i++) {
-			depthMap[i] = (float) UintdepthMap[i] / 1000.0f;
-		}
-		free(UintdepthMap);
-		return res;
-	}
-
-	inline uint2 getinputSize() {
-		return _size;
-	}
+      if (total != expected_size) {
+        std::cout << "End of file" << (total == 0 ? "" : "(garbage found)")
+          << "." << std::endl;
+        return false;
+      } else {
+        return true;
+      }
+    }
+
+    /**
+     * Restart the reader from the beginning.
+     */
+    inline void restart() {
+      _frame = -1;
+      _pose_num = -1;
+      rewind(_pFile);
+      if (_gt_file.is_open())
+        _gt_file.seekg(0, _gt_file.beg);
+    }
+
+    /**
```

Comment:
This comment doesn't add any information wrt the function signature.

Reply:
Removed in d0e0cf4dcde9e2be2904002f599c9a285638de8c.